apps/frontend/src/app/Components/Character/CharacterCard.tsx

```diff
@@ -379,43 +373,43 @@ function HeaderContent() {
         color={characterEle}
         sx={{ opacity: 0.85 }}
       />
-      <Grid container spacing={1} flexWrap="nowrap">
-        <Grid item sx={{ textShadow: '0 0 5px gray' }}>
+      <Box display="flex" gap={1} sx={{ textShadow: '0 0 5px gray' }}>
+        <Box>
           <Typography component="span" variant="h6" whiteSpace="nowrap">
             Lv. {characterLevel}
           </Typography>
           <Typography component="span" variant="h6" color="text.secondary">
             /{ascensionMaxLevel[ascension]}
           </Typography>
-        </Grid>
-        <Grid item>
-          <Typography variant="h6">
-            <SqBadge>C{constellation}</SqBadge>
-          </Typography>
-        </Grid>
-      </Grid>
-      <Grid container spacing={1} flexWrap="nowrap">
-        <Grid item>
-          <Chip
-            color={autoBoost ? 'info' : 'secondary'}
-            label={<strong>{tAuto}</strong>}
-          />
-        </Grid>
-        <Grid item>
-          <Chip
-            color={skillBoost ? 'info' : 'secondary'}
-            label={<strong>{tSkill}</strong>}
-          />
-        </Grid>
-        <Grid item>
-          <Chip
-            color={burstBoost ? 'info' : 'secondary'}
-            label={<strong>{tBurst}</strong>}
-          />
-        </Grid>
-      </Grid>
-      <Typography mt={1}>
-        <StarsDisplay stars={characterSheet.rarity} colored />
+        </Box>
+        <Typography
+          component="span"
+          variant="h6"
+          whiteSpace="nowrap"
+          color={`roll${constellation < 3 ? 3 : constellation}.main`}
+        >
+          C{constellation}
+        </Typography>
+      </Box>
+      <Box display="flex" gap={1}>
+        <Chip
+          size="small"
+          color={autoBoost ? 'info' : 'secondary'}
+          label={<strong>{tAuto}</strong>}
+        />
+        <Chip
+          size="small"
+          color={skillBoost ? 'info' : 'secondary'}
+          label={<strong>{tSkill}</strong>}
+        />
+        <Chip
+          size="small"
+          color={burstBoost ? 'info' : 'secondary'}
+          label={<strong>{tBurst}</strong>}
+        />
+      </Box>
+      <Typography variant="h6" lineHeight={1}>
+        <StarsDisplay stars={characterSheet.rarity} colored inline />
```

Comment:
Stars are small again

Reply:
Stars are small again to make space, it was occupying a lot of vertical space.